tests/unit/utils/scheduler/test_schedule.py

```diff
@@ -450,12 +447,49 @@ def test_handle_func_check_data(self):
         }
 
         with patch("salt.utils.process.daemonize"), patch("sys.platform", "linux2"):
-            # run handle_func once
-            self.schedule.handle_func(False, "test.arg", data)
+            with patch.object(self.schedule, "standalone", return_value=True):
```

Comment:
Are we sure `self.schedule.standalone` is not set to a mock instead of returning `True`.
You probably need to do `with patch.object(self.schedule, "standalone", True):` instead...
Let's make sure.